packages/react-components/lib/lifts/lift-table.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
It seems that this view is missing the functionality to open the lift request dialog.